src/client/client4.js

```diff
@@ -1097,6 +1101,19 @@ export default class Client4 {
         );
     };
 
+    uploadBrandImage = async (imageData) => {
```

Comment:
I don't think so since there's only the one attachment, not multiple fields like in the regular file upload. It works for the webapp as is